### ISSUE TITLE
Integrate blog posts on website

### ISSUE THREAD

**Issue by uekerman** [precice/precice.github.io#402]
Blog posts live on Discourse. Currently we link from the button "blog" in the header to the news category on Discourse.
It would be nice if we could integrate the content directly on the website somehow without duplication.
Could also be important for SEO?

Connected to #146 ?

**Comment by MakisH** [comment]
I am not sure if integrating the whole post on the website would help, but I see the argument of SEO because of having new content, or because of including (even) more content. But I think that linking for each post to Discourse shows where the community lives.

Looking at other examples, I think that Ubuntu maintains a [blog on their website](https://ubuntu.com/blog) (more oriented towards customers) and a [separate announcements category on their Discourse](https://discourse.ubuntu.com/c/announcements/12). That's probably too much for us.

How about a small section displaying the 3-4 latest titles from Discourse, between the "Welcome to preCICE" and the "Prepared for the..."? Would show some activity, without overloading too much (more).

**Comment by uekerman** [comment]
> How about a small section displaying the 3-4 latest titles from Discourse, between the "Welcome to preCICE" and the "Prepared for the..."? Would show some activity, without overloading too much (more).

Sounds good. You mean to manually curate this section then, right?

**Comment by MakisH** [comment]
> You mean to manually curate this section then, right?

No, I meant a dynamic list that shows the newest threads using the [Discourse API](https://docs.discourse.org/#tag/Topics/operation/listLatestTopics).

**Comment by MakisH** [comment]
Similarly to #584, we could have a section in the landing page, between `Ready-to-use with your favourite open-source solver` and `What our community is saying` (maybe in light blue, like other sections), that would display 3-4 cards from the [News category](https://precice.discourse.group/c/news/5). No further buttons/filters would be needed, just a link to the category.

cc @bytepaul

**Comment by MakisH** [comment]
Thanks to @BytePaul (#611), we now have the news on the landing page:

<img width="1285" height="572" alt="Image" src="https://github.com/user-attachments/assets/cc9d0ccd-10da-4322-84b3-459dd1f72ce8" />

Potential improvements:

- The sorting and date field are currently based on the "last activity". A more interesting point would be the date published. Discourse provides a "day created", but this might not be the same as the one published, as we often draft news topics on a private category, before publishing. Most of the time, the "last activity" is indeed the interesting field, since the further activity is typically also further news.
- The date field uses no locale and, at least on my browser, it picks the `en-us`, showing the month before the day. One could easily set `en-gb`, for example. Unless this is indeed using the user's locale.
- The heading is centered, but since this also includes the anchor symbol, it is not perfectly centered when the symbol is hidden. One could come up with a creative solution here (or just align left, again). While on it, we could also switch the heading to sentence case.
- The title field cannot accommodate a third line, but we already have topics with such long titles (which we should generally avoid).

**Comment by uekerman** [comment]
- Currently doesn't work on my phone (cards don't get stacked)
- "Read more about this update" could be dropped if the complete card is clickable
- I think we could increase the ratio "content / space". I would like to see more of the summary.

**Comment by BytePaul** [comment]
Hi @uekerman ,

I have updated the CSS layout file to work on Mobile/Tablets. 

The changes are not merged yet.

Here is the preview : 

<img width="481" height="857" alt="Image" src="https://github.com/user-attachments/assets/d8322ba7-d104-4915-a746-0e9fd3efbf69" />

**Comment by MakisH** [comment]
Addressed most of the issues in #628. Remaining:

- [x] Date used for sorting (created/last activity, ideally we want a "published on")
- [ ] Centering of the section heading (not that important)
- [x] Longer titles -> But we should not have such long titles

**Comment by MakisH** [comment]
Feedback by @carme-hp and @uekerman:

- [x] Let's move the news section higher up. I would suggest making it the second section (after "welcome preCICE"), since we should have a balance between visible news (returning visitors) and quick description of what the project does (new visitors).

**Comment by Wdhanashri** [comment]
Submitted a PR in #732 addressing the remaining sorting item from #628 news cards now sort by publication date (created_at) instead of last activity. Also moved the date inside the clickable area so the entire card is a link.
